### PR TITLE
Update-5-request-payment-details.txt

### PR DESCRIPTION
**Pull request by degeri**
Adds options for already registed users and users with multiple payouts per month.

### CODE REVIEW

File: 2019/email-templates/5-request-payment-details.txt [hunk 1]
```diff
@@ -1,42 +1,105 @@
...
+    Proposal Token=d33a2667469b56942adf42453def6cc2292325251e4cf791e806939ea9efc9e1
+    Expense (USD)=<XXX>
+{End LOOP}
 
+    The total displayed should be {TOTAL USD}
 
-Please note this website is for production use. Do not attempt test/find vulnerabilities or try to submit false/test invoices. Any such action might lead to suspension of your account.
+    Once its filled hit "Submit"
 
+    You can then head over to https://cms.decred.org/user/invoices and ensure that your invoice has been submitted.Your payment will be made by the end of the current month.
```
Comment (jholdstock) [inline 2019/email-templates/5-request-payment-details.txt]:
```suggestion
    You can then head over to https://cms.decred.org/user/invoices and ensure that your invoice has been submitted. Your payment will be made by the end of the current month.
```

File: 2019/email-templates/5-request-payment-details.txt [hunk 2]
```diff
@@ -1,42 +1,105 @@
...
 
-Month should be <MONTH> and Year should be <YEAR>. 
+    You will received an email with the subject "Welcome to the Contractor Management System" follow through on the link and register an account within 24 hours of receiving the email. 
 
-"Contractor name"=[Full Name]
-"Contractor location" = [Your Location]
-"Contractor contact" = [Your email]
-"Contractor rate (USD)" = 5
-Payment address = [Your Decred Address]
+    Then click on "submit invoice"
```
Comment (jholdstock) [inline 2019/email-templates/5-request-payment-details.txt]:
```suggestion
    Then click on "Submit Invoice" 
```

File: 2019/email-templates/5-request-payment-details.txt [hunk 3]
```diff
@@ -1,42 +1,105 @@
...
-If you would like to be listed on our hall of fame, please provide us with the name you would like to appear and you may choose to provide one link to your social media, github or blog.  
+{LOOP AS NEEDED}
+    You are eligible for a payment of <XXX> USD in DCR for issue {Link to issue}.
+{END LOOP}
 
-You will received an email with the subject "Welcome to the Contractor Management System" follow through on the link and register an account within 24 hours of receiving the email. 
+    {if non pgp: Please note payment was reduced by X USD due to your email not being encrypted with PGP]
 
-Then click on "submit invoice" 
+    If you would like to be listed on our hall of fame, please provide us with the name you would like to appear and you may choose to provide one link to your social media, github or blog.
```
Comment (jholdstock) [inline 2019/email-templates/5-request-payment-details.txt]:
```suggestion
    If you would like to be listed on our hall of fame, please provide us with the name you would like to appear and you may choose to provide one link to your social media, GitHub or blog.  
```

File: 2019/email-templates/5-request-payment-details.txt [hunk 4]
```diff
@@ -1,42 +1,105 @@
...
+{END LOOP}
 
-You will received an email with the subject "Welcome to the Contractor Management System" follow through on the link and register an account within 24 hours of receiving the email. 
+    {if non pgp: Please note payment was reduced by X USD due to your email not being encrypted with PGP]
 
-Then click on "submit invoice" 
+    If you would like to be listed on our hall of fame, please provide us with the name you would like to appear and you may choose to provide one link to your social media, github or blog.  
 
-Month should be <MONTH> and Year should be <YEAR>. 
+    You will received an email with the subject "Welcome to the Contractor Management System" follow through on the link and register an account within 24 hours of receiving the email.
```
Comment (jholdstock) [inline 2019/email-templates/5-request-payment-details.txt]:
```suggestion
    You will receive an email with the subject "Welcome to the Contractor Management System" follow through on the link and register an account within 24 hours of receiving the email. 
```

File: 2019/email-templates/5-request-payment-details.txt [hunk 5]
```diff
@@ -1,42 +1,105 @@
...
 
+    The total displayed should be {TOTAL USD}
 
-Please note this website is for production use. Do not attempt test/find vulnerabilities or try to submit false/test invoices. Any such action might lead to suspension of your account.
+    Once its filled hit "Submit"
 
+    You can then head over to https://cms.decred.org/user/invoices and ensure that your invoice has been submitted.Your payment will be made by the end of the current month.
 
-If you need support creating a Decred wallet or address, you can visit our documentation website or you can can ask for help on our support channels.
+    Please note this website is for production use. Do not attempt test/find vulnerabilities or try to submit false/test invoices. Any such action might lead to suspension of your account.
```
Comment (jholdstock) [inline 2019/email-templates/5-request-payment-details.txt]:
```suggestion
    Please note this website is for production use. Do not attempt to test/find vulnerabilities or try to submit false/test invoices. Any such action might lead to suspension of your account.
```

File: 2019/email-templates/5-request-payment-details.txt [hunk 6]
```diff
@@ -1,42 +1,105 @@
...
+
+    We are happy to inform you are now eligible for payment.
+
+{LOOP AS NEEDED}
+    You are eligible for a payment of <XXX> USD in DCR for issue {Link to issue}.
+{END LOOP}
+
+    {if non pgp: Please note payment was reduced by X USD due to your email not being encrypted with PGP]
+
+    If you would like to be listed on our hall of fame, please provide us with the name you would like to appear and you may choose to provide one link to your social media, github or blog.
```
Comment (jholdstock) [inline 2019/email-templates/5-request-payment-details.txt]:
```suggestion
    If you would like to be listed on our hall of fame, please provide us with the name you would like to appear and you may choose to provide one link to your social media, GitHub or blog.  
```

File: 2019/email-templates/5-request-payment-details.txt [hunk 7]
```diff
@@ -1,42 +1,105 @@
...
+
+{LOOP AS NEEDED}
+    You are eligible for a payment of <XXX> USD in DCR for issue {Link to issue}.
+{END LOOP}
+
+    {if non pgp: Please note payment was reduced by X USD due to your email not being encrypted with PGP]
+
+    If you would like to be listed on our hall of fame, please provide us with the name you would like to appear and you may choose to provide one link to your social media, github or blog.  
+
+    You are alredy registed in the decred CMS system. Login to your account.
```
Comment (jholdstock) [inline 2019/email-templates/5-request-payment-details.txt]:
```suggestion
    You are already registered in the Decred CMS system. Login to your account. 
```

File: 2019/email-templates/5-request-payment-details.txt [hunk 8]
```diff
@@ -1,42 +1,105 @@
...
+    You are eligible for a payment of <XXX> USD in DCR for issue {Link to issue}.
+{END LOOP}
+
+    {if non pgp: Please note payment was reduced by X USD due to your email not being encrypted with PGP]
+
+    If you would like to be listed on our hall of fame, please provide us with the name you would like to appear and you may choose to provide one link to your social media, github or blog.  
+
+    You are alredy registed in the decred CMS system. Login to your account. 
+
+    Then click on "submit invoice"
```
Comment (jholdstock) [inline 2019/email-templates/5-request-payment-details.txt]:
```suggestion
    Then click on "Submit Invoice" 
```

File: 2019/email-templates/5-request-payment-details.txt [hunk 9]
```diff
@@ -1,42 +1,105 @@
...
+{End LOOP}
+
+    The total displayed should be {TOTAL USD}
+
+    Once its filled hit "Submit"
+
+    You can then head over to https://cms.decred.org/user/invoices and ensure that your invoice has been submitted. Your payment will be made by the end of the current month.
+
+
+    Please note this website is for production use. Do not attempt test/find vulnerabilities or try to submit false/test invoices. Any such action will lead to suspension of your account.
```
Comment (jholdstock) [inline 2019/email-templates/5-request-payment-details.txt]:
```suggestion
    Please note this website is for production use. Do not attempt to test/find vulnerabilities or try to submit false/test invoices. Any such action will lead to suspension of your account.
```

**Comment by jholdstock** [comment]
Just for future info, you can actually accept a reviewers suggestions in a batch.

This will create a single commit which includes all of the suggested changes, rather than a commit per change (as above)

**Comment by degeri** [comment]
Guess it dint show that option on the mobile site

**Pull request merged**